modules/tracker/models/pdo/SubmissionModel.php

```diff
@@ -259,17 +259,13 @@ public function getLatestSubmissionByProducerDateAndBranch($producerDao, $date =
         $dayBeforeQueryTime = date('Y-m-d H:i:s', strtotime($queryTime) - self::SEC_IN_DAY);
         $sql = $this->database->select()->setIntegrityCheck(false)
             ->from('tracker_submission')
-            ->join(
-                'tracker_scalar',
-                'tracker_submission.submission_id = tracker_scalar.submission_id',
-                array())
-            ->where('tracker_submission.submit_time <= ?', $queryTime);
+            ->where('submit_time <= ?', $queryTime);
         if ($onlyOneDay) {
-            $sql = $sql->where('tracker_submission.submit_time > ?', $dayBeforeQueryTime);
+            $sql = $sql->where('submit_time > ?', $dayBeforeQueryTime);
         }
-        $sql = $sql->where('tracker_submission.producer_id = ?', $producerDao->getKey())
+        $sql = $sql->where('producer_id = ?', $producerDao->getKey())
             ->where('branch = ?', $branch)
-            ->order('tracker_submission.submit_time', 'DSC')
+            ->order('submit_time', 'DSC')
```

Comment:
I fixed this on another branch, but it's probably best to fix it here also.

https://github.com/midasplatform/Midas/pull/221/commits/f9b8565f8f1102935f24a623e5fdc16075b122b7